src/lib/taggingUtils.ts

```diff
@@ -63,15 +63,23 @@ const taggingUtils = {
     return Object.keys(tagObject).map(tagKey => this.buildTag(tagKey, tagObject[tagKey]));
   },
 
-  generateCustomTags(annotationList: string[] = []): string[] {
-    return annotationList.map(el => this.buildTag(ANNOTATION_LABEL, el));
+  generateCustomTags({
```

Comment:
Why did you chose to hand in the params as an object? Wouldn't useFallback be a good second parameter? Then the standard case would stay generateCustomTags(annotations).

Reply:
Oh, I think that was a mistake on my part. I think I somehow thought that this would break if the list contains more than one string, but if it's passed as an array (as it was), that's obviously not actually an issue.

Reply:
Implemented in https://github.com/d4l-data4life/js-sdk/pull/45/commits/e7f38dcedccbcf0ddcdee5e47b0bb7de4d5f4a54